### ISSUE TITLE
Homepage #TeamCloudNative section v2

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#571]
See figma for updates to design here with some icons and Phippy characters. Also add transition for when shapes change to new people. See screenshot below:
<img width="851" alt="Screen Shot 2022-12-04 at 8 03 29 AM" src="https://user-images.githubusercontent.com/463160/205501733-f705d2c6-ab66-4138-864a-4d8e825dc1c2.png">

**Comment by thetwopct** [comment]
Waiting on revised details of HOCN #579

**Comment by cjyabraham** [comment]
#579 is actually an independent issue. This issue just needs clear design direction from @GarethAcidWorks as there are multiple designs now in figma.

**Comment by thetwopct** [comment]
#579 incorporated the HOCN heads in to the TeamCloudNative section

**Comment by cjyabraham** [comment]
Correct. And that has nothing to do with this issue. I've updated the description above with a screenshot to make it clear what design updates I was referring to.

**Comment by thetwopct** [comment]
Sure. But the updates requirements have changed since creation of the issue to incorporate HOCN in to this section.

**Comment by GarethAcidWorks** [comment]
@KTan1226 Let's add this into this weeks call please.

It's been flagged as an outstanding issue.

Thanks

**Comment by KTan1226** [comment]
Hi @GarethAcidWorks - this is on hold until OKRs for 2023 are determined. Thanks!

**Comment by cjyabraham** [comment]
Icons have been swapped in. Smooth transitions are still probably impossible.
